src/options.ts

```diff
@@ -458,3 +464,43 @@ export function normalizeRouteRules(
   }
   return normalizedRules;
 }
+
+function _resolveExportConditions(
+  conditions: string[] = [],
+  opts: { dev: boolean; node: boolean }
+) {
+  const resolvedConditions: string[] = [];
+
+  // 1. Add dev or production
+  resolvedConditions.push(opts.dev ? "development" : "production");
```

Comment:
I think we should add development but not production @pi0

Reply:
It is valid https://nodejs.org/api/packages.html#packages_community_conditions_definitions

Reply:
Usecase is also for nested conditions. `exports: { production: { node, import, worker }, development: { node, worker, import }`. Conditions are checked from first to last and can be combined

Reply:
Sorry, I meant we should add it for node, not for worker-like environments. But I haven't seen this key used too much in the wild, so this should be fine.